modules/database/src/tools/DBD/Output.pm

```diff
@@ -42,12 +42,13 @@ sub OutputDBD {
 
 sub OutputDB {
     my ($out, $dbd) = @_;
-    OutputRecords($out, $dbd->records);
+    OutputRecords($out, $dbd);
 }
 
 sub OutputMenus {
     my ($out, $menus) = @_;
-    while (my ($name, $menu) = each %{$menus}) {
+    foreach my $name (sort keys %{$menus}) {
+        my $menu = $menus->{$name};
```

Comment:
As you know, my perl is sketchy at best.  Is this iterating a hash after sorted order?

Does this mean that `-s` effects only record instances?  (as well as COMMENT and POD)

Reply:
DBD files are always sorted, DB files it's optional.

Approved.